src/main/telemetry/hott.c

```diff
@@ -105,7 +110,7 @@ static uint8_t hottMsgCrc;
 #define HOTT_CRC_SIZE (sizeof(hottMsgCrc))
 
 #define HOTT_BAUDRATE 19200
-#define HOTT_PORT_MODE MODE_RXTX // must be opened in RXTX so that TX and RX pins are allocated.
+#define HOTT_INITIAL_PORT_MODE MODE_RX
```

Comment:
This will break in some cases with dynamic resource allocation. Please revert this part.